pkg/domain/infra/abi/containers.go

```diff
@@ -552,7 +553,17 @@ func (ic *ContainerEngine) ContainerTop(ctx context.Context, options entities.To
 
 	// Run Top.
 	report := &entities.StringSliceReport{}
-	report.Value, err = container.Top(options.Descriptors)
+	descriptors := []string{}
+	for _, arg := range options.Descriptors {
+		if strings.Contains(arg, " ") {
+			// Quote multi work args
+			descriptors = append(descriptors, fmt.Sprintf("%q", arg))
+		} else {
+			descriptors = append(descriptors, arg)
+		}
+	}
```

Comment:
this makes no sense to me as well. On the local path we can juts pass down the array there is no need for this string manipulation.

Reply:
This issue is passing -o "foo bar" is valid for PS.  We need to maintain the "" in this call to pass it down to ps command.